docs/team/bryanhce.md

```diff
@@ -38,14 +39,14 @@ details.
 * **Documentation**:
   * User Guide:
     * Did final checks to ensure consistency of UG (pull request #142).
-    * Add glossary (pull request #129).
+    * Added the glossary section (pull request #129).
     * Updated introduction and subheadings (pull request #148).
-    * Update add guest in user guide (pull request #122).
+    * Updated add guest in user guide (pull request #122).
```

Comment:
Do you want to have code block here for `add` to be clearer?

Reply:
Fixed!